functions/Get-DbaDbBackupHistory.ps1

```diff
@@ -429,7 +429,7 @@ function Get-DbaDbBackupHistory {
                     if ((Get-PSCallStack)[1].Command -notlike ' Get-DbaDbBackupHistory*') {
                         $forkCheckSql = "
                             SELECT
-                                database_name,
+                                database_name as 'name',
```

Comment:
Can you make the alias `databasename` instead?

Or simply  update the referenced property name to be `database_name` instead of it using `name`.